sktree/stats/tests/test_forestht.py

```diff
@@ -205,12 +209,12 @@ def test_linear_model(hypotester, model_kwargs, n_samples, n_repeats, test_size)
                     n_jobs=-1,
                 ),
                 "random_state": seed,
-                "permute_per_tree": True,
-                "sample_dataset_per_tree": True,
+                "permute_per_tree": False,
```

Comment:
This is testing MI Sep and not MI/Tree right? Where are we testing MI/Tree?

Reply:
This is MI / tree

Reply:
Agreed. This is MI Sep. We should test MI / Tree as well.

Reply:
Yeah I will add a pytest.mark.parametrize in a bit. I think last night the pvalue behavior was not converging as well as the MI Sep.